.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -14,7 +14,7 @@ Please understand this diagram before proceeding. It explains how to get your PR
 What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!
 
   - [ ] New resource provider.
-  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
+  - [X] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
```

Comment:
You shouldn't be editing this template as part of your change.

Reply:
Oh oops, do I create a new document with the .md? I was unsure how to use the template per the above commenter

Reply:
These should go in your PR description. It looks like you didn't select the template at PR creation time. You will want to create a new PR and select the correct template as the description says.

Reply:
@weshaggard Hi! I've tried making a new PR but each time at creation it doesn't use the template. If I can leave a comment with the same information is that sufficient? If not, could you maybe write down the steps to see if I'm doing something wrong?

Reply:
Before you hit the create button did you click the preview tab in the description and then click the control plane template link? It should reload your PR with the correct template and then you click the create button.